node/blockAPI/shardBlock.go

```diff
@@ -111,3 +117,55 @@ func (sbp *shardAPIBlockProcessor) convertShardBlockBytesToAPIBlock(hash []byte,
 		Status:          BlockStatusOnChain,
 	}, nil
 }
+
+func (sbp *shardAPIBlockProcessor) setStatusESDTTransferTransactionsCrossShard(miniblocks []*api.MiniBlock) {
+	for _, mb := range miniblocks {
+		if mb.Type != block.TxBlock.String() {
+			continue
+		}
+
+		// ignore miniblocks that are not cross shard destination me
+		if mb.SourceShard == mb.DestinationShard || mb.DestinationShard != sbp.selfShardID {
+			continue
+		}
+
+		iterateMiniblockTxsForESDTTransfer(mb, miniblocks)
+	}
+}
+
+func iterateMiniblockTxsForESDTTransfer(miniblock *api.MiniBlock, miniblocks []*api.MiniBlock) {
+	for _, tx := range miniblock.Transactions {
+		if !strings.HasPrefix(string(tx.Data), core.BuiltInFunctionESDTTransfer) {
+			continue
+		}
+
+		searchUnsignedTransaction(tx, miniblocks)
+	}
+}
+
+func searchUnsignedTransaction(tx *transaction.ApiTransactionResult, miniblocks []*api.MiniBlock) {
+	// search for unsigned transactions
```

Comment:
useless comment as the function is called searchUnsignedTransaction

Reply:
removed.